### PR TITLE
Prevent Importing expired/undigitised items into show plans.

### PR DESCRIPTION
**Pull request by mstratford**
Something that should have been there from the start (this has caused lots of people to play old jingles etc on shows because they continuously imported them from show to show).

Tested (and still running) on myradio-dev.

### CODE REVIEW

File: src/Public/js/nipsweb.import.js [hunk 1]
```diff
@@ -133,45 +134,56 @@ function updateTimeslotList() {
...
-      for (item in data.payload[selectedChannelNo]) {
-        if (data.payload[selectedChannelNo][item].type == "central") {
-          if (!data.payload[selectedChannelNo][item].clean) {
-            cleanStars = "**";
-          } else {
-            cleanStars = "";
-          }
-          extraString = " - " + data.payload[selectedChannelNo][item].artist + " - " + data.payload[selectedChannelNo][item].album.title + " (" + data.payload[selectedChannelNo][item].length + ")";
-          itemid = data.payload[selectedChannelNo][item]["album"].recordid + "-" + data.payload[selectedChannelNo][item].trackid;
+      currentShowPlan = data.payload;
```
Comment (LordAro) [inline src/Public/js/nipsweb.import.js]: Global state :(
How feasible would it be to make this a parameter for loadChannelList ?

Comment (mstratford) [reply src/Public/js/nipsweb.import.js]: Not sure what to here, apart from reloading the show plan AJAX every time I change channel (which I fixed in this PR from previous behavior)

**Comment by mstratford** [comment]
Fixes: #767

File: src/Public/js/nipsweb.import.js (hunk 1 above)
Comment (Brookke) [reply src/Public/js/nipsweb.import.js]: Without creating all the channels and then dynamically hiding and showing them, I think this is as good as we are going to get

Comment (mstratford) [reply src/Public/js/nipsweb.import.js]: Yeah.

**Pull request merged**